src/css-files/ResourceCategory.css

```diff
@@ -0,0 +1,24 @@
+#angular-checkbox + label:before{
+    border: 0.2rem solid #000;
+    border-radius: 0.2rem;
+    display: inline-block;
+    width: 2rem;
+    height: 2rem;
+    margin-right: 0.2rem;
+}
+
+/* #angular-checkboxes:checked + label:before {
+    background-color: MediumSeaGreen;
+} */
+
+.angular-label, .react-label, .git-label, .js-label, .css-label, .python-label {
```

Comment:
instead of declaring the same styles for multiple selectors can you please use one generic selector apply it to every element that needs the style